paginator/is_nil_test.go

```diff
@@ -0,0 +1,53 @@
+package paginator
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_IsNil(t *testing.T) {
```

Comment:
[minor]

I think we could rename this file name to `paginator_utils_test.go` to keep the consistency between other test files.

And this test case maybe could be called `TestIsNil` to conform to golang naming convention.